<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>расчет для:</string>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Атакая</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Защита</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Доход</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Расчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сумма :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Купить</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>211</width>
      <height>461</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Попрошайка</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Крестьянин</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Шут</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Палач</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Монах</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Алхимик</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Сквайр</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Торговец</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Рыцарь</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Епископ</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Лорд</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Кузнец</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Кораблестроитель</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Морской страж</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Есть сейчас</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="background">
      <color>
       <red>0</red>
       <green>160</green>
       <blue>0</blue>
      </color>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>0</string>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="10" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="11" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="12" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="13" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>321</width>
      <height>461</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Попрошайка</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Крестьянин</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Шут</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Палач</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Монах</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Алхимик</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Сквайр</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Торговец</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Рыцарь</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Епископ</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Лорд</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Кузнец</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Кораблестроитель</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Морской страж</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Нужно купить</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="background">
      <color>
       <red>0</red>
       <green>160</green>
       <blue>0</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Будет в итоге</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>0</string>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="9" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="10" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="10" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="11" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="11" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="12" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="12" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="13" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="13" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Бонус :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Текущие значения:</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>63</width>
       <height>146</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Атака:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Защита:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Доход:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Есть сейчас</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Нужно докупить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>720</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Строка для копирования  *Итога :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>740</y>
      <width>561</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
